stream-chat-android-ui-components/src/main/res/values-es/strings_message_list.xml

```diff
@@ -27,6 +27,7 @@
     <item quantity="one">"%d Respuesta"</item>
     <item quantity="other">"%d Respuestas"</item>
   </plurals>
+  <string name="stream_ui_message_audio_reply_info">"Mensaje de voz"</string>
```

Comment:
We should only update "default" language, the rest of them should be downloaded from localazy